app/workers/evss/disability_compensation_form/submit_form0781.rb

```diff
@@ -31,6 +31,21 @@ class SubmitForm0781 < Job
         Metrics.new(STATSD_KEY_PREFIX, msg['jid']).increment_exhausted
       end
 
+      def get_docs(submission_id, uuid)
+        @submission_id = submission_id
+        parsed_forms = JSON.parse(submission.form_to_json(Form526Submission::FORM_0781))
+        ret_paths = []
```

Comment:
Perhaps the `ret_paths` var name could be more descriptive?

Same with `f` and `c` - the params in the `each` block.

Feel free to ignore. Just my preference.

Reply:
This is good input and I appreciate it. Even I didn't remember what I was actually doing here because of the crappy named variables. I have also added comments to the method. 

Updated in 95e09b829eca560cc76df0c01083b9c01ba46af2